lib/utils/matchUrl.js

```diff
@@ -20,14 +20,14 @@ exports.isUriIncluded = function(includePattern, uri) {
     //     test = true;
     // }
     // else {
-    return Object.keys(includePattern).some(function (key) {
-        // console.log(includePattern[key]);
-        // @todo includePattern[key] can throw an error if user passes a whitespace
+    return Object.keys(includePatterns).some(function (key) {
+        // console.log(includePatterns[key]);
+        // @todo includePatterns[key] can throw an error if user passes a whitespace
         //       in front of *.url and it fails silently at the moment.
-        var pattern = new MatchPattern(includePattern[key]);
+        var pattern = new MatchPattern(includePatterns[key]);
 
         // use val
-        // console.log('pattern = ' + includePattern[key]);
+        // console.log('pattern = ' + includePatterns[key]);
         return ( pattern.test(uri) );
```

Comment:
nit-pick: no spaces after ( and before ). In this case, they are not needed at all.